src/go/k8s/kuttl-test.yaml

```diff
@@ -8,6 +8,6 @@ testDirs:
   - ./tests/e2e
 kindConfig: ./kind.yaml
 commands:
-  - command: "kubectl apply -k config/default"
+  - command: "kubectl apply -k config/e2e-test"
```

Comment:
in PR #665 we use make deploy, this won't work

Reply:
Thank you! I have changed that to `make deploy`